src/renderers/shared/ReactPerf.js

```diff
@@ -344,14 +372,18 @@ function getMeasurementsSummaryMap(measurements) {
 }
 
 function start() {
+  returnWarnIfDevFalse();
   ReactDebugTool.beginProfiling();
 }
 
 function stop() {
+  returnWarnIfDevFalse();
+  alreadyWarned = false;
```

Comment:
Oh, I see your point now. I propose we always warn just once. You can potentially leave `start()` and `stop()` in some hot spot and we don’t want to spam the logs.